test/qpy_compat/test_qpy.py

```diff
@@ -1077,10 +1077,11 @@ def load_qpy(qpy_files, version_parts):
 
     from qiskit.qpy.exceptions import QpyError
 
-    while pulse_files:
-        path, version = pulse_files.popitem()
+    for path, min_version in pulse_files.items():
```

Comment:
Oh thanks for updating this based on my comments in the earlier PR. I didn't expect that.